### ISSUE TITLE
Issue notification a configurable time interval before the event start

### ISSUE THREAD

**Issue by tbabej** [nicoe/remhind#1]
First of all, thanks for this great project. It's a much needed addition to the opensource ecosystem.

Currently, remhind issues a notification for events without alarms explicitly set once that event starts. Since in practice, majority of the .ics files do not come with alarms embedded, it would be more helpful if the user could (perhaps on a calendar-level base) define a default time interval (i.e. 5 minutes) that remhind would notify him before next event start:

```
9.55: Event "Meeting with the boss" starts in 5 minutes.
```

instead of the default behaviour of

```
10.00: Event "Meeting with the boss" started
```

this would allow for some time to find a meeting room, move between buildings, find a cell phone number to call, etc :)

**Comment by nicoe** [comment]
Hello ! Thanks for your appreciation.

I am already planning to have a "remind hour" configuration for events which due date is a date and not a datetime (because the RFC specify it should be 00:00 UTC but it's not practical). I might as well add one for this use case (it should be a list of minutes I think so that multiple reminders are possible, negative values would be after the due date).

But I must say that I am used to add alarms to my events and thus did not thought that people would not use those.

**Comment by tbabej** [comment]
Yeah! To provide more context on that, it could happen if you're working in a team setting and

a) you're sharing a team calendar
b) calendar invites are sent to your email

basically any scenario when you are not in the control of creating the calendar event. You _could_ go over them manually, but you can imagine that with a busy calendar that becomes quite an ordeal.

I fully agree with the proposed implementation.
